.gitignore

```diff
@@ -0,0 +1,9 @@
+lerna-debug.log
+node_modules
+.serverless
+package-lock.json
```

Comment:
Is there a reason we're ignoring this lockfile? I assume to just use the yarn lock?

Reply:
Removed! + Improved the .gitignore by borrowing some from https://github.com/github/gitignore/blob/master/Node.gitignore.